internals-js/src/tagSpec.ts

```diff
@@ -24,14 +24,15 @@ export class TagSpecDefinition extends FeatureDefinition {
     this.printedTagDefinition = 'directive @tag(name: String!) repeatable on FIELD_DEFINITION | INTERFACE | OBJECT | UNION';
     if (!this.isV01()) {
       this.tagLocations.push(
+        DirectiveLocation.SCHEMA,
```

Comment:
This will cause the definition to change for `0.2` in addition to `0.3`.

Instead, you'll want to:
- Define a function called `isV02()` on this class, similar to `isV01()`.
- Inside this `if` block, at the end of it, add another `if` block checking `!this.isV02()`.
- Inside that `if` block, call `this.tagLocations.push()` with `DirectiveLocation.SCHEMA`.
- After that `push()` call, set `this.printedTagDefinition` to the one you have below, with the additional `SCHEMA` location.